.github/workflows/workflow.yml

```
@@ -99,10 +99,10 @@ jobs:
       - name: Init variables
         id: init_variables
         run: |
-          if [[ github.ref == 'refs/heads/master' ]]
+          if [[ "${{  github.ref }}" == "refs/heads/master" ]]
           then
             TAG='master'
-          elif [[ startsWith(github.ref, 'refs/tags/') ]]
+          elif [[ "${{  github.ref }}" == "refs/tags/*" ]]
```

Using glob syntax (the `*`) in bash tests (the `[[` and `]]`) won't work I believe.

I did try the following in my `bash` prompt and it doesn't _echo_ `No`.

```bash
if [[ "/ref/123" == "/ref/*" ]]; then echo Yes; else echo No; fi
```

You can try a regular expression match, like the following (not tested)

```suggestion
          elif [[ "${{  github.ref }}" =~ refs/tags/.* ]]
```
Note the intentional absence of quotes for the regular expression. The following does _echo_ `Yes` in `bash.

```bash
if [[ "/ref/123" =~ /ref/.* ]]; then echo Yes; else echo No; fi
```